docs/programming_guide.md

```diff
@@ -2398,6 +2399,61 @@ of a function, or use the `withParameters(...)` method to pass in a configuratio
 [Back to top](#top)
 
 
+Passing parameters to functions
+-------------------
+
+Parameters can be passed to rich functions using either the constructor (if the function is
```

Comment:
ok, so which requirements do exist for the operator to be serializable? or can i just omit those details and say something like "parameters can be stored in non-transient fields if the function is serializable".